### ISSUE TITLE
Remove inconsistent themeing in Client

### ISSUE THREAD

**Issue by pranavtifr** [longturn/freeciv21#762]
**Describe the bug**
Client has certain places where the theme is hardcoded from an earlier era and it leads to inconsistencies.

**To Reproduce**
I\ll give one such example of such an inconsistency. We can add more to this thread to keep track maybe.
- Try to Quit Client
You see this, 
![image](https://user-images.githubusercontent.com/30164619/144764486-abee1c39-a295-481c-aec7-788b4c7b1121.png)

**Expected behavior**
This should be consistent with the theme of the client, this is from previous nightstalker theme

**Platform and version (please complete the following information):**
 - OS: All
 - Freeciv21 version: 3.0-alpha9 (dd5bebe52b4cab581fa8e9d3144962c72ce033ab)

**Additional context**
Maybe requires refactoring some client code.

**Comment by lmoureaux** [comment]
So I suppose this is about themes other than Nightstalker?

**Comment by lmoureaux** [comment]
`Classic` is different from NS:
![image](https://user-images.githubusercontent.com/22327575/144783365-05011225-10af-4deb-8893-3d13d7267fec.png)

**Comment by pranavtifr** [comment]
> So I suppose this is about themes other than Nightstalker?

Yes. It is fixed in some places.. Like quit in classic and necrophos, but the quit in thelastlasermaster is still with nightstalker. There are other inconsistencies in other popups like buy confirmation. 

Also, it has a fixed font which cant be changed ? That needs to be fixed too. It needs to have the same default font.

**Comment by jwrober** [comment]
This is what all of them look like next to each other
![image](https://user-images.githubusercontent.com/23037938/150701507-a7b71001-7ee6-4014-b89d-10f45973f20c.png)

I don't really see an issue here.

**Comment by lmoureaux** [comment]
I think we should polish one dark and one light and drop the rest

**Comment by jwrober** [comment]
That is probably classic for light and nightstalker for dark. I think that is a great idea.

**Comment by lmoureaux** [comment]
I think I see what the original issue was about (@pranavtifr if you recall): after *changing* the theme, it looks inconsistent. If you restart it looks fine.

**Comment by lmoureaux** [comment]
Related to #1502
